onelogin/client.go

```diff
@@ -245,13 +229,9 @@ func (c *Client) VerifyFactor(token string, p *VerifyFactorParams) (*VerifyFacto
 }
 
 // NewClient creates a new Client and returns a pointer to it.
-func NewClient() *Client {
-	return &Client{
-		Endpoints: Endpoints{
-			GenerateSamlAssertion: GenerateSamlAssertionURL,
-			GenerateTokens:        GenerateTokensURL,
-			GetUserByEmail:        GetUserByEmailURL,
-			VerifyFactor:          VerifyFactorURL,
-		},
-	}
+func NewClient(region string) (c *Client, err error) {
+	c.Endpoints = Endpoints{Region: region}
```

Comment:
This line causes a panic because `c` is nil.
I suggest changing the line to: `c = &Client{Endpoints: Endpoints{Region: region}}`

Reply:
Ah- I see. I don't know how I didn't hit that, I think there's a missing test on `NewClient`.